src/App.js

```diff
@@ -25,7 +28,11 @@ class App extends Component {
         release_date: movie.release_date,
       };
     });
-    this.setState({ movies: [...moviesData] });
+    this.setState({ movies: [...moviesData], selectedMovie: null });
```

Comment:
Is there a reason you're using the spread operator here? Why not `movies: moviesData` without the spread operator?